src/server/tests/gutters/gutterSelection.ts

```diff
@@ -0,0 +1,135 @@
+import { inCountryGroups } from '../../../shared/lib';
+import {
+    GutterTargeting,
+    GutterTest,
+    GutterTestSelection,
+    GutterVariant,
+    UserDeviceType,
+} from '../../../shared/types';
+
+import { selectVariantUsingMVT } from '../../lib/ab';
+import { audienceMatches, correctSignedInStatus, pageContextMatches } from '../../lib/targeting';
+
+import { TestVariant } from '../../lib/params';
+
+const moduleName = 'Gutter';
+
+// hard coded tests
```

Comment:
I think it might be best to avoid hardcoded tests if we can help it. It avoids ancient copy turning up in the future - see https://github.com/guardian/support-dotcom-components/pull/1256

Reply:
Thanks Tom - removed.